internal/services/compute/orchestrated_virtual_machine_scale_set_data_source.go

```diff
@@ -133,7 +133,7 @@ func (r OrchestratedVirtualMachineScaleSetDataSource) Attributes() map[string]*p
 			},
 		},
 
-		"identity": commonschema.SystemAssignedIdentityComputed(),
+		"identity": commonschema.SystemAssignedUserAssignedIdentityComputed(),
```

Comment:
can we also update the documentation to reflect this?

Reply:
The docs appear to be update to date for this? https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/orchestrated_virtual_machine_scale_set#identity